ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/SponsoredInteraction.java

```diff
@@ -0,0 +1,108 @@
+package com.mozilla.telemetry.contextualservices;
+
+import com.google.auto.value.AutoValue;
+import org.apache.beam.sdk.coders.*;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessageWithAttributesCoder;
+import org.apache.beam.sdk.values.TypeDescriptor;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.Map;
+
+@AutoValue
+public abstract class SponsoredInteraction implements Serializable {
+    // topsites, suggest
+    public static final String SOURCE_TOPSITE = "topsites";
+    public static final String SOURCE_SUGGEST = "suggest";
```

Comment:
The document types for desktop suggest use `quicksuggest-` as a prefix rather than `suggest`, reflecting previous proposed branding of the feature. We probably want `quicksuggest` here.